.ci/magician/cmd/test_terraform_vcr.go

```diff
@@ -292,7 +292,7 @@ func modifiedPackages(newBranch, oldBranch string, rnr ExecRunner) (map[string]s
 	}
 	var goFiles []string
 	for _, line := range strings.Split(diffs, "\n") {
-		if strings.HasSuffix(line, ".go") || line == "go.mod" || line == "go.sum" {
+		if strings.HasSuffix(line, ".go") || strings.Contains(line, "test-fixtures") || line == "go.mod" || line == "go.sum" {
```

Comment:
I see that the bash version of this is matching against `go.mod$` - why is that? Should we use HasSuffix here?

Reply:
I don't think there's any particular reason other than that the other pattern ends in `$`. We don't have any other files that end with `go.mod` or `go.sum`.

Reply:
Oh, never mind. I forgot the line contains the full path of the file.